src/pf-dropdown/pf-dropdown.spec.js

```diff
@@ -24,7 +24,34 @@ describe("Patternfly Dropdown Component Test", function () {
 
     customElement.addEventListener('hidden.bs.dropdown', function () {
       expect(button.parentNode.classList.contains('open')).toBe(false);
-      done();
     }, false);
   });
+
+  it("disabled button should not open dropdown", function () {
+    var button = customElement.querySelector('.btn');
+    button.classList.add('disabled');
+    customElement.toggle();
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+    button.dispatchEvent(new MouseEvent('click'));
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+  });
+
+  it("should select an element", function () {
+    var buttonText = customElement.querySelector('.btn');
+    var item = customElement.querySelector('ul.dropdown-menu > li:nth-child(2)');
+    expect(buttonText.innerText).toBe('Dropdown');
+    expect(item.innerText).toBe('Item 1');
+  });
+
+  it("should not click disabled item", function () {
```

Comment:
click() does not behave like a real mouse click on the anchor tag. I had to create a new mouse event(cancelable: true) to test the preventDefault() which prevents the URL from changing. And also it's easy to expect event.defaultPrevented on this new event. What do you think?